robot-server/robot_server/deletion_planner.py

```diff
@@ -70,6 +70,8 @@ def plan_for_new_protocol(
         Args:
             existing_protocols: Information about all protocols that currently exist.
                 Must be in order from oldest first!
+            exclude_kind: Exclude these kids  of protocols when taking into account
```

Comment:
```suggestion
            exclude_kind: Exclude these kinds of protocols when taking into account
```

Typo comment correction.